types/k6/experimental/browser.d.ts

```diff
@@ -1131,6 +1148,37 @@ export interface ElementHandle extends JSHandle {
      */
     selectText(options?: ElementHandleOptions): void;
 
+    /**
+     * Sets the value of the file input to these files.
+     *
+     * To work with local file on the file system, work with the experimental
+     * fs module to load and read the file's content.
+     *
+     * The elementHandle must an [input element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
+     * @param files
+     * @param options
+     */
+    setInputFiles(files: File | File[], options?: {
+        /**
+         * Maximum time in milliseconds. Pass 0 to disable the timeout. Default
+         * is overridden by the setDefaultTimeout option on `BrowserContext` or
+         * `Page`. Defaults to 30000.
```

Comment:
Should we link these, too? (`@link ...`)

Reply:
Added it in 6867bb061121b7ba780b49a15a2024903c331b62